app/models/reports/generated_report.rb

```diff
@@ -5,5 +5,9 @@
 module Reports
   class GeneratedReport < ActiveRecord::Base
     self.table_name = :defra_ruby_exporters_bulk_export_files
+
+    def starts_from
+      Date.parse(file_name.split("-").first, :plain_year_month_day)
```

Comment:
This is a TODO fix (this introduces a constraint in the file name format that needs reinforcement). Is good to go for an initial dev env test though and the refactoring is almost ready so I would like to merge this in the meantime